app-devices/yubioath-flutter/autobuild/prepare

```
@@ -0,0 +1,19 @@
+# Adapted from AUR: yubico-authenticator
+#
+# Ref: https://aur.archlinux.org/cgit/aur.git/tree/?h=yubico-authenticator&id=aa91641d83d6c7e8274543bb03ae9a26c7c434a9
+
+abinfo "Setting up flutter ..."
+export $(grep '^FLUTTER=' "$SRCDIR/.github/workflows/env")
+fvm install $FLUTTER
+fvm global $FLUTTER
+
+abinfo "Disabling flutter analytics ..."
+fvm flutter --disable-analytics
+
+abinfo "Removing helper installation from CMakeLists ..."
+sed -i '/build\/linux\/helper/d' linux/CMakeLists.txt
```

This should be a patch, sed(1) fails silently.